src/components/presentational/InfiniteScroller/InfiniteScroller.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useRef, useState } from 'react'
+import { useInterval } from '../../../hooks';
+
+export interface InfiniteScrollerProps {
+	onTrigger: Function;
+	enabled: boolean;
+}
+
+export default function (props: InfiniteScrollerProps) {
```

Comment:
So this "InfiniteScroller" actually doesn't do infinite scrolling but rather just runs a function when it's visible

Feels like it makes sense to change the name of this to OnVisibleTrigger to more explicitly describe what it's doing?